internal/dinosaur/pkg/workers/dinosaurmgrs/dinosaurs_auth_config_mgr.go

```diff
@@ -109,8 +114,23 @@ func augmentWithStaticAuthConfig(r *dbapi.CentralRequest, centralConfig *config.
 
 // augmentWithDynamicAuthConfig performs all necessary rituals to obtain auth
 // configuration via RHSSO API.
-func augmentWithDynamicAuthConfig(_ *dbapi.CentralRequest, _ *config.CentralConfig) error {
-	// TODO(alexr): Talk to RHSSO dynamic client API.
+func augmentWithDynamicAuthConfig(r *dbapi.CentralRequest, centralConfig *config.CentralConfig, iam sso.IAMService) error {
+	host := r.GetUIHost()
+	if host == "" {
+		return fmt.Errorf("cannot create OIDC client when no host provided for %s/%s", r.Namespace, r.Name)
+	}
+
+	name := fmt.Sprintf("rhacs-central-idp-%s", r.Name)
```

Comment:
```suggestion
	name := fmt.Sprintf("acsms-%s", r.Name)
```

Per [openapi spec](https://sso.dev.redhat.com/auth/realms/redhat-external/apis/openapi.yaml), name is limited to 50 chars, so let's be (1) frugal with extras and (2) truncate the name in rhsso client to avoid unnecessary `400`s.